src/views/accompagnement/liste.vue

```diff
@@ -227,14 +227,17 @@ export default {
         ? `/api/followups/id/${this.$route.params.surveyId}`
         : `/api/followups/surveys`
       try {
-        const response = await fetch(uri, {
-          method: "GET",
-          redirect: "error",
-          headers: {
-            Accept: "application/json",
-            "Content-Type": "application/json",
-          },
-        })
+        const response = await fetch(
+          `${uri}?${Math.random().toString(36).substr(2, 5)}`,
+          {
+            method: "GET",
+            redirect: "error",
+            headers: {
+              Accept: "application/json",
+              "Content-Type": "application/json",
+            },
+          }
+        )
```

Comment:
N'est-il pas possible de configurer avec axios comme dans cet exemple

Si non un commentaire me parait adapté pour comprendre cet ajout.
Une variante que j'apprécie peut être plus :

```suggestion
        const response = await fetch(
          `${uri}?${Math.random().toString(36).substr(2, 5)}`,
          {
            method: "GET",
            params: {
             // Ajoute un date pour éviter la mise en cache du serveur
              t: new Date().getTime()
            }
            redirect: "error",
            headers: {
              Accept: "application/json",
              "Content-Type": "application/json",
            },
          }
        )
```

Reply:
> N'est-il pas possible de configurer avec axios comme dans cet exemple
> 
> Si non un commentaire me parait adapté pour comprendre cet ajout. Une variante que j'apprécie peut être plus :

Effectivement c'est plus propre comme ça. 
Le but de la PR est peut être pas hyper clair, mais j'ai l'impression que sous certaines conditions le navigateur de l'utilisateur mets en cache l'API (et la redirection vers l'auth github), ce qui empêche le chargement normal de la page de l'outil d'accompagnement